test/modules/fundingManager/bondingCurve/utils/mocks/FM_BC_Bancor_Redeeming_VirtualSupply_NativeV1Mock.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import "forge-std/console.sol";
+
+import {
+    FM_BC_Bancor_Redeeming_VirtualSupply_v1,
+    IFM_BC_Bancor_Redeeming_VirtualSupply_v1,
+    FM_BC_Tools
+} from "@fm/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_v1.sol";
+
+import {FM_BC_Bancor_Redeeming_VirtualSupply_Native_v1} from
+    "@fm/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_Native_v1.sol";
+
+contract FM_BC_Bancor_Redeeming_VirtualSupply_NativeV1Mock is
+    FM_BC_Bancor_Redeeming_VirtualSupply_Native_v1
+{
+    function call_reserveRatioForBuying() external view returns (uint32) {
```

Comment:
we call these "exposed_" now :grin: Check out the repo CoC: https://docs.google.com/document/d/10jYXETE6ySx1e1yLOSY7Ccfy52Mfee1OJUEqGv-_wSA/edit

Reply:
updated to `exposed_`